### DISCUSSION TITLE
Is it posible to define jobs in a setup sperate from the app that embeds quartz?

### DISCUSSION THREAD

**Discussion by LB-personal** [quartznet/quartznet#1062] [Q&A] [answered]
Hello all,
I have a scenario in which I want to define the jobs for a scheduler before the app that instantiate it runs, probably from an installer.

Is this possible?

I have tried publishing the JobStore scheme into sql express database, inserting a job and a simple trigger into it by sql script and then run a test app that instantiate a scheduler with an AdoJobStore connected to the DB mentioned above.
The scheduler ignored the job I defined.  is there something obvious I am missing?

**Comment by lahma** [comment, marked as answer]
You can also setup the XML file and it will update your job store to that state. [Here's the XML job/trigger configuration example](https://github.com/quartznet/quartznet/tree/master/src/Quartz.Examples/10_RunningJobsByPlugInXmlConfiguration).
